projects/plugins/jetpack/extensions/blocks/map/controls.js

```diff
@@ -60,38 +60,6 @@ export default ( {
 		}
 	};
 
-	/**
-	 * Change event handler for the map height sidebar control. Ensures the height is valid,
-	 * and updates both the height attribute, and the map component's height in the DOM.
-	 *
-	 * @param {Event} event - The change event object.
-	 */
-	const onHeightChange = event => {
```

Comment:
I might be missing something, but I'm not sure this is needed. I couldn't observe anything weird without this callback, which eventually just updates the `mapHeight` attribute.